articles/azure-functions/durable/durable-functions-entities.md

```diff
@@ -80,15 +110,50 @@ public class Counter
+
 > [!NOTE]
 > The function entry point method with the `[FunctionName]` attribute *must* be declared `static` when using entity classes. Non-static entry point methods may result in multiple object initialization and potentially other undefined behaviors.
 
-In the class-based programming model, the `IDurableEntityContext` object is available in the `Entity.Current` static property.
+### The Entity Context Object
 
-The class-based model is similar to the programming model popularized by [Orleans](https://www.microsoft.com/research/project/orleans-virtual-actors/). In this model, an entity type is defined as a .NET class. Each method of the class is an operation that can be invoked by an external client. Unlike Orleans, however, .NET interfaces are optional. The previous *Counter* example did not use an interface, but it can still be invoked via other functions or via HTTP API calls.
+Entity-specific functionality can be accessed via a context object of type `IDurableEntityContext`. This context object is available as a parameter (for the function-based syntax) or via the async-local property `Entity.Current`.
+
+The following members are used to signal other entities, or start new orchestrations:
+
+* **SignalEntity(EntityId, operation, input)**: sends a one-way message to an entity.
+* **CreateNewOrchestration(orchestratorFunctionName, input)**: starts a new orchestration.
+
+The following members provide general information about the currently executing entity.
+
+* **EntityName**: the name of the currently executing entity.
+* **EntityKey**: the key of the currently executing entity.
+* **EntityId**: the ID of the currently executing entity (includes name and key).
+
+The following members are meant for use with the function-based syntax. 
+
+* **OperationName**: the name of the current operation.
+* **GetInput\<TInput>()**: gets the input for the current operation.
+* **Return(arg)**: returns a value to the orchestration that called the operation.
+
+The following members manage the state of the entity (create, read, update, delete). They are not usually needed when using the class-based syntax, though it makes sense to call them explicitly in some situations (for example, to delete the entity state).
+
+* **HasState**: whether the entity exists, that is, has some state. 
+* **GetState\<TState>()**: gets the current state of the entity. If it does not already exist, it is created.
+* **SetState(arg)**: creates or updates the state of the entity.
+* **DeleteState()**: deletes the state of the entity. 
+
+If the state returned by `GetState` is an object, it can be directly modified by the application code. There is no need to call `SetState` again at the end (but also no harm). If `GetState<TState>` is called multiple times, the same type must be used.
```

Comment:
This is reference information and probably belongs in the **durable-functions-bindings.md** article. Conceptual articles like these are intended to be as language-neutral as possible with the exception of code-samples to help make the concepts more concrete.